src/main/java/net/glowstone/entity/meta/MetadataType.java

```diff
@@ -19,8 +19,8 @@
     ITEM(ItemStack.class),
     BOOLEAN(Boolean.class),
     VECTOR(EulerAngle.class),
-    POSITION(Position.class),
```

Comment:
Well I'm not really sure why `Position.class` was used here, that class is a utility and is not initializable. Location on the other hand can hold position data, so I think this is the most appropriate type.

[This](https://github.com/GlowstoneMC/Glowstone/pull/374/files/0444a35caece2595b33296ab2faa0d8c55e2a8a7#diff-270aed41c5e1297cb3ddafcfdf4efeebR150) is where the Location data is actually transformed into the correct [format](http://wiki.vg/Protocol#Position).